18F/C2
# Finish the ahoy_matey configuration

`user_id` needs to be saved in `visits` table, and possibly more. This is one task from [this Trello card](https://trello.com/c/cOC2CLEk/445-analytics-updates-capture-desired-ux-metrics-with-current-system-analytics-google-com-prod-console-or-by-new-means). Was previously this PR #1303.

To Test
----------

1. Browse to the app locally
2. In the browser's web console, execute `ahoy.reset();` to force a new Visit.
3. Look at the `visits` and `ahoy_events` tables.
4. Verify that visits have user_id's. E.g.: `Visit.first.user` should return your user.
